pkg/cointicker/cointicker.go

```diff
@@ -55,23 +60,25 @@ func decorateRatio(r, lr float64, c CoinTicker) func(interface{}) aurora.Value {
 	return c.au.Red
 }
 
-// Results ...
-type Results struct {
-	Result map[string]float64
-}
+func (c CoinTicker) generateResult() (results storage.Results, errors []error) {
```

Comment:
Maybe you could not use named returns here and create a `Results` element from scratch (perhaps even using a `NewResults` type of function) in the method.

Reply:
Also returning from `cointicker` something that is defined in `storage` may not be so untangled as you'd like.